src/constants.js

```diff
@@ -9,15 +9,20 @@ const GITHUB_REPOSITORY = process.env["GITHUB_REPOSITORY"];
 
 const REPO_DIRECTORY = process.env["GITHUB_WORKSPACE"];
 
+const JPEG_QUALITY = process.env["INPUT_JPEGQUALITY"] ? process.env["INPUT_JPEGQUALITY"] : 80;
+const PNG_QUALITY = process.env["INPUT_PNGQUALITY"] ? process.env["INPUT_PNGQUALITY"] : 80;
+const WEBP_QUALITY = process.env["INPUT_WEBPQUALITY"] ? process.env["INPUT_WEBPQUALITY"] : 80;
+const IGNORE_PATHS = process.env["INPUT_IGNOREPATHS"] ? process.env["INPUT_IGNOREPATHS"].split(",") : ["nodemodules/**"];
```

Comment:
Can we use the format of:

`process.env['KEY'] || process.env['INPUT_KEY']` instead?

Reply:
yes, we can.